upgrade/capsule.py

```diff
@@ -129,7 +129,7 @@ def satellite6_capsule_upgrade(cap_host, sat_host):
     # Host and Content Host are in different organizations (e.g. host not in
     # org, and content host in one), the content host will be unregistered as
     # part of the upgrade process.
-    if to_version in ['6.2', '6.3', '6.4']:
+    if to_version in ['6.2', '6.3', '6.4', '6.5']:
```

Comment:
```suggestion
    if float(to_version) >=6.2:
```
Not mandatory as we are going to remove this everything soon.